src/main/resources/META-INF/resources/primefaces-extensions/fab/fab-widget.js

```diff
@@ -25,8 +25,13 @@ PrimeFaces.widget.ExtFAB = PrimeFaces.widget.BaseWidget.extend({
                 document.selection.empty();
             }
         };
-        var that = this;
-        this.button.click(function(){ that.toggle(); });
+        var obj = this;
+        this.button.click(function(){ obj.toggle(); });
+        if (cfg.keepOpen === undefined) {
```

Comment:
I think you can say "if (!cg.keepOpen)" which will evaluate if its FALSE or UNDEFINED?

Reply:
Correct!